elasticmock/fake_elasticsearch.py

```diff
@@ -216,7 +216,7 @@ def suggest(self, body, index=None, params=None):
             raise NotFoundError(404, 'IndexMissingException[[{0}] missing]'.format(index))
 
         result_dict = {}
-        for key, value in body.iteritems():
+        for key, value in body.items():
```

Comment:
`items` work on both Python3 and Python2. Considering the fact, that body is never that big, it's OK to use just `items`.

https://stackoverflow.com/questions/10458437/what-is-the-difference-between-dict-items-and-dict-iteritems